lib/cn/hblock_builder.c

```diff
@@ -212,37 +235,48 @@ hbb_finish(
     if (num_kblocks == 0 && !wbb_entries(bld->ptree))
         return 0;
 
-    /* Header, HyperLogLog */
-    /* [HSE_TODO]: vblk idx adjust */
+    min_pfxp = min_pfxp ? : &min_pfx;
+    max_pfxp = max_pfxp ? : &max_pfx;
```

Comment:
I believe the idiom is to write it as ?: with no intervening white space, but as tristan noted this is a GNU-ism and we should avoid it in new code and fix it in old code.  I think maybe clang will warn on this???